ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/SelectorsPollerScript.js

```diff
@@ -84,7 +90,14 @@ window.__firefox__.execute(function($) {
     pendingOrigins: new Set(),
     // A map of origin strings and their isFirstParty results
     urlFirstParty: new Map(),
-    alreadyKnownFirstPartySubtrees: new WeakSet()
+    alreadyKnownFirstPartySubtrees: new WeakSet(),
+    // All the procedural rules that exist and need to be processed
+    // when the script is loaded and a new element is added
+    proceduralActionFilters: undefined,
+    // Tells us if procedural filtering is available
+    hasProceduralActions: false,
+    // The map of style attributes and their values used for procedural filtering
+    styleAttrMap: new Map()
```

Comment:
appears to be unused?

Reply:
ahh yes remained from converting the desktop script